### ISSUE TITLE
Windows 11 virus

### ISSUE THREAD

**Issue by gabriele-v** [moneymanagerex/moneymanagerex#6388]
MMEX version:
 - [ ] 1.6.4
 - [x] 1.7.0
 - [ ] 1.7.1
 - [ ] 1.7.2
 - [ ] Other (please specify)

Note: bug reporters are expected to have verified the bug still exists
either in the last stable version of MMEX or on updated development code
(master branch).

Operating System:
 - [x] Windows
 - [ ] Mac OSX
 - [ ] Linux 

# Description of the bug
mmex.exe is recognized as virus from Windows 11 and it's automatically removed from the system

# Reproduction

Is the bug reproducible? 
 - [x] Always 
 - [ ] Randomly 
 - [ ] Happened only once

# Additional information

![image](https://github.com/moneymanagerex/moneymanagerex/assets/7106290/51293e0d-144c-4ec3-991a-d6f50f12804e)

**Comment by Dandy64** [comment]
I have been using the portable version of Money Manager Ex - 1.64. Windows 11 is installed on my computer and I do not have any issues.

But after this post, I have taken mmex.exe and uploaded it to https://www.virustotal.com/ (I highly recommend this site, by the way!)

No problems were found.
![VirusTotal](https://github.com/moneymanagerex/moneymanagerex/assets/83161661/25042fc8-8ecc-4529-827a-d3fd683a530a)

**Comment by whalley** [comment]
Yes it's a false positiive... We have had them before.

**Comment by gabriele-v** [comment]
Maybe we should consider to sign the exe?

**Comment by pdelamotar** [comment]
It has happened to me before and I have reported it here too but in other compilations it does not happen, I still sent a request to recognize it as not suspicious (Firefox considers it suspicious at the time of finishing the download)

**Comment by Dandy64** [comment]
> It has happened to me before and I have reported it here too but in other compilations it does not happen, I still sent a request to recognize it as not suspicious (Firefox considers it suspicious at the time of finishing the download)

Yeah, and I know why it is happening. See the comment of [gabriele-v]. The file is not signed.

**Comment by whalley** [comment]
https://github.com/moneymanagerex/moneymanagerex/issues/2355
